etc/nebula-storaged.conf.default

```diff
@@ -76,6 +76,14 @@
 #   * kAll, Collect all stats
 --rocksdb_stats_level=kExceptHistogramOrTimers
 
+# Whether or not to enable rocksdb's prefix bloom filter, disabled by default.
+--enable_prefix_filtering=false
```

Comment:
For the option name format, pls start with `rocksdb_` like others.

Reply:
Since HBase also support prefix-bloom-filter, I think we'd better make it as a general option, WDYT?

Reply:
If so maybe you should remove the `rocksdb` from comment

Reply:
Make sense. But we don't plan to support hbase again.

Reply:
Changed to enable_rocksdb_prefix_filtering